### PR TITLE
Tech sub

### PR DESCRIPTION
**Pull request by ekzyis**
I wanted to learn more about how subs are implemented and afaict, this is all that is needed to add new subs.

I tested creating new posts but I didn't test the RSS feed since I have no trusted user to put an item on the front page.

https://github.com/stackernews/stacker.news/assets/27162016/974ee8d2-b6f5-48fb-9220-22e6a3acb143

_creating a link post in tech sub (sorry for slow page loads, haha)_

edit: Okay, I was able to test the RSS feed using this patch:

```diff
diff --git a/pages/~/[sub]/rss.js b/pages/~/[sub]/rss.js
index 4380b5d..be495b3 100644
--- a/pages/~/[sub]/rss.js
+++ b/pages/~/[sub]/rss.js
@@ -5,5 +5,5 @@ import { ITEMS } from '../../../fragments/items'
 export default function RssFeed () {
   return null
 }
-
-export const getServerSideProps = getGetRssServerSideProps(ITEMS)
+const variables = { sort: 'recent' }
+export const getServerSideProps = getGetRssServerSideProps(ITEMS, variables)
```

 and it works

### CODE REVIEW

File: components/footer.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ const handleThemeChange = (dark) => {
 
 const RssPopover = (
   <Popover>
-    <Popover.Content style={{ fontWeight: 500, fontSize: '.9rem' }}>
+    <Popover.Content className='d-flex' style={{ fontWeight: 500, fontSize: '.9rem' }}>
```
Comment (ekzyis) [inline components/footer.js:83]: This was needed to fix this:

2023-06-12-120757_298x115_scrot

Now looks like this:

2023-06-12-122651_290x91_scrot

**Comment by ekzyis** [comment]
Some thoughts:

- The RSS feed selection won't scale with more subs (like user subs)
- The helper text won't scale. It's already pretty weird now:
```html
<li>If it's bitcoin related, put it in the bitcoin sub.</li>
<li>If it's nostr related, put it in the nostr sub.</li>
<li>If it's tech related, put it in the tech sub.</li>
<li>If it's a job, put it in the jobs sub.</li>
```

**Comment by ekzyis** [comment]
> You're not using the scrubbed db I sent you?

I didn't load it into my new setup yet

**Comment by huumn** [comment]
> Some thoughts:
> 
> * The RSS feed selection won't scale with more subs (like user subs)
> * The helper text won't scale. It's already pretty weird now:
> 
> ```
> <li>If it's bitcoin related, put it in the bitcoin sub.</li>
> <li>If it's nostr related, put it in the nostr sub.</li>
> <li>If it's tech related, put it in the tech sub.</li>
> <li>If it's a job, put it in the jobs sub.</li>
> ```

Agreed, >5 subs and all of our sub related UX needs to change.

**Pull request merged**